pkg/api/const.go

```diff
@@ -26,10 +26,12 @@ const (
 // the LinuxDistros supported by vlabs
 const (
 	Ubuntu          Distro = "ubuntu"
+	Ubuntu1804      Distro = "ubuntu-18.04"
```

Comment:
default distro doesn't change, we are simply adding a two new distro options for new deployments. Upgrades and scale operations are unaffected.